tasks/rslang/english-for-kids-translated.md

```diff
@@ -0,0 +1,153 @@
+- afterwards the application automatically redirects to the main page with a list of categories
+
+### Hacker scope +40
+
+- **Statistics Page: (+40)**
+  - [ ]  statistics page contains a list of all categories, all words in each category, and a translation of each word. The minimum width at which statistics page is displayed correctly is 320 px. The presence of a scrollbar in a table with statistics is not considered to be an error: (+10)
+  - [ ] statistics is displayed next to each word - how many times a card with a given word was clicked in training mode, how many times this word was guessed in game mode, how many mistakes were made, the percentage of correct answers for each word in game mode. After restarting the application, statistics are saved: (+10)
+  - [ ] it is possible to sort the data alphabetically, for numerical data ​​- by their value. Sorting can occur in direct and reverse order and should cover the entire data range: (+10)
+  - [ ] there are "Repeat difficult words" and "Reset" buttons on statistics page. The "Reset" button resets statistics. Clicking on the "Repeat difficult words" button opens a page similar to the category page with most frequent words guessed wrong. The page "Repeat difficult words" can have from zero to eight words, depending on how many words were guessed wrong in play mode. After clicking on the "Reset" button, the number of words on the "Repeat difficult words" page is set to zero: (+10)
+  - [ ] additional functionality outside of the scope of the task is implemented. This part is not evaluated, but if you can do better, then why not.
+
+## Penalty Points
+- [ ] less than eight categories, less than eight words in each category, data for the last two categories is copied from the demo, and not prepared independently: -10 points
+- [ ]  requirements for pull request, repository, and commit names are not met: -10 points (evaluated by mentors only)
+- [ ] code requirements are not met: -10 points or more (evaluated by mentors only)
+
+## Video explaining the task description
+- Task overview: https://youtu.be/xXBce4ZLcCQ
+- Task Q & A: https://youtu.be/ctpBIPygV4k
+- Task review https://youtu.be/PdHjLvcOLQw
+
+## Resources
+- [Folder with data](./english-for.kids.data)  for six categories of words (American pronunciation is used). For the other two missing categories, you should collect the data independently using Internet resources
+- [Bootstrap themes](https://bootswatch.com/)
+- [Localstorage](https://learn.javascript.ru/localstorage)
+
+### Information resources
+- [ru.forvo.com](https://ru.forvo.com/) - get English pronunciation of the word (several pronunciation options are offered)
+- [dictionary.cambridge.org](https://dictionary.cambridge.org/) - get English pronunciation of the word
+- [wooordhunt.ru](https://wooordhunt.ru/word/welcome) - get English pronunciation of the word
+
+### Cross-check
```

Comment:
The resources aren't translated. I'd consider removing the section as I'm not sure whether the process of the cross-check is needed